src/main/resources/reference.conf

```diff
@@ -20,12 +20,12 @@ resourceTypes =
     },
     {
       name = "billing-project"
-      actions = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "launch_notebook"]
+      actions = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "launch_notebook", "list_notebooks"]
```

Comment:
nitpick: can we call these launch_notebook_clusters and list_notebook_clusters? or is it too late because these are populated?

Reply:
We can still change them!